stockflux-securities-master/src/components/Button.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import "./Button.css";
+import PropTypes from "prop-types";
+
+const Button = ({ text, size, className, ...props }) => {
+  return (
+    <div className={`button-container ${size} ${className}`}>
```

Comment:
Why are you putting a `div` around the button here? Why not just add the `className` prop onto the `button` JSX element directly?

Reply:
I was copying what I had done before but yeah, I've changed it to this way now